gunicorn_conf.py

```diff
@@ -0,0 +1,5 @@
+"""Gunicorn production config."""
+import multiprocessing
+
+bind = '127.0.0.1:5000'
+workers = multiprocessing.cpu_count() * 2 + 1
```

Comment:
Shouldn't we just set it to 3? Seems massively overkill for an application that does close to nothing... And pretty annoying when running this locally and getting 17 (!) workers + master.